src/App.css

```diff
@@ -0,0 +1,144 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+* {
+    box-sizing: border-box;
+}
+
+html{
+    max-width: 600px;
+    margin: 0 auto;
+}
+
+body{
+    background-color: lightblue;
+}
+
+.todoList{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    flex-direction: row;
+    width:100%;
+}
+
+header{
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+    align-items: center;
+}
+
+form{
+    display: flex;
+}
+
+form > input{
+    border: none;
+    background-color: lightblue;
+    border-right: 10px;
+    border-bottom: 1px solid red;
+    outline: none;
+    margin-right: 10px;
+}
+input::placeholder{
+    color:red;
+}
+
+.todo-btn{
+    color:white;
+    padding: 10px;
+    background-color: red;
+    border: 1px solid red;
+}
+
+
+.todo{
+    display: flex;
+    justify-content: center;
+    background-color: red;
+    color:white;
+    align-items: center;
+    margin: 15px 0;
+    width: 45%;
+    height: 30px;
+}
+
+.completed{
+    display: flex;
+    justify-content: center;
+    color:white;
+    align-items: center;
+    margin: 15px 0;
+    width: 45%;
+    height: 30px;
+    background-color: blue;
+    text-decoration: line-through;
+}
+
+h3{
+    display: none;
+}
+h1{
+    font-size: 40px;
+    margin: 20px 0;
+    color: red;
+}
+
+button{
+    cursor: pointer;
+    color:white;
+    padding: 10px;
+    background-color: red;
+    border: 1px solid red;
+}
+
+.clear{
+    margin: 20px 0;
+}
+
```

Comment:
You might want to rename this class to `.clear-btn` (for clarity and consistency with your other button-specific classes).